src/main/resources/com/google/api/codegen/metadatagen/java/grpc/grpc_package.snip

```diff
@@ -127,7 +72,7 @@
           }
 
           pom.project {
-            name "com.google.api.grpc:{@metadata.identifier}"
+            name "com.google.api.grpc:grpc-{@metadata.identifier}"
```

Comment:
Don't compose the package name in the snippet, pass in the full name through the model (other places too)